modules/SofaMiscFem/src/SofaMiscFem/QuadBendingFEMForceField.inl

```diff
@@ -0,0 +1,1261 @@
+  Kb[1][15]=Kb0[1][15]+Kb1[1][15]+Kb2[1][15]+Kb3[1][15];
+  Kb[1][16]=Kb0[1][16]+Kb1[1][16]+Kb2[1][16]+Kb3[1][16];
+
+  Kb[5][0]=Kb0[5][0]+Kb1[5][0]+Kb2[5][0]+Kb3[5][0];
+  Kb[5][1]=Kb0[5][1]+Kb1[5][1]+Kb2[5][1]+Kb3[5][1];
+  Kb[5][5]=Kb0[5][5]+Kb1[5][5]+Kb2[5][5]+Kb3[5][5];
+  Kb[5][6]=Kb0[5][6]+Kb1[5][6]+Kb2[5][6]+Kb3[5][6];
+  Kb[5][10]=Kb0[5][10]+Kb1[5][10]+Kb2[5][10]+Kb3[5][10];
+  Kb[5][11]=Kb0[5][11]+Kb1[5][11]+Kb2[5][11]+Kb3[5][11];
+  Kb[5][15]=Kb0[5][15]+Kb1[5][15]+Kb2[5][15]+Kb3[5][15];
+  Kb[5][16]=Kb0[5][16]+Kb1[5][16]+Kb2[5][16]+Kb3[5][16];
+
+  Kb[6][0]=Kb0[6][0]+Kb1[6][0]+Kb2[6][0]+Kb3[6][0];
+  Kb[6][1]=Kb0[6][1]+Kb1[6][1]+Kb2[6][1]+Kb3[6][1];
+  Kb[6][5]=Kb0[6][5]+Kb1[6][5]+Kb2[6][5]+Kb3[6][5];
+  Kb[6][6]=Kb0[6][6]+Kb1[6][6]+Kb2[6][6]+Kb3[6][6];
+  Kb[6][10]=Kb0[6][10]+Kb1[6][10]+Kb2[6][10]+Kb3[6][10];
+  Kb[6][11]=Kb0[6][11]+Kb1[6][11]+Kb2[6][11]+Kb3[6][11];
+  Kb[6][15]=Kb0[6][15]+Kb1[6][15]+Kb2[6][15]+Kb3[6][15];
+  Kb[6][16]=Kb0[6][16]+Kb1[6][16]+Kb2[6][16]+Kb3[6][16];
+
+  Kb[10][0]=Kb0[10][0]+Kb1[10][0]+Kb2[10][0]+Kb3[10][0];
+  Kb[10][1]=Kb0[10][1]+Kb1[10][1]+Kb2[10][1]+Kb3[10][1];
+  Kb[10][5]=Kb0[10][5]+Kb1[10][5]+Kb2[10][5]+Kb3[10][5];
+  Kb[10][6]=Kb0[10][6]+Kb1[10][6]+Kb2[10][6]+Kb3[10][6];
+  Kb[10][10]=Kb0[10][10]+Kb1[10][10]+Kb2[10][10]+Kb3[10][10];
+  Kb[10][11]=Kb0[10][11]+Kb1[10][11]+Kb2[10][11]+Kb3[10][11];
+  Kb[10][15]=Kb0[10][15]+Kb1[10][15]+Kb2[10][15]+Kb3[10][15];
+  Kb[10][16]=Kb0[10][16]+Kb1[10][16]+Kb2[10][16]+Kb3[10][16];
+
+  Kb[11][0]=Kb0[11][0]+Kb1[11][0]+Kb2[11][0]+Kb3[11][0];
+  Kb[11][1]=Kb0[11][1]+Kb1[11][1]+Kb2[11][1]+Kb3[11][1];
+  Kb[11][5]=Kb0[11][5]+Kb1[11][5]+Kb2[11][5]+Kb3[11][5];
+  Kb[11][6]=Kb0[11][6]+Kb1[11][6]+Kb2[11][6]+Kb3[11][6];
+  Kb[11][10]=Kb0[11][10]+Kb1[11][10]+Kb2[11][10]+Kb3[11][10];
+  Kb[11][11]=Kb0[11][11]+Kb1[11][11]+Kb2[11][11]+Kb3[11][11];
+  Kb[11][15]=Kb0[11][15]+Kb1[11][15]+Kb2[11][15]+Kb3[11][15];
+  Kb[11][16]=Kb0[11][16]+Kb1[11][16]+Kb2[11][16]+Kb3[11][16];
+
+  Kb[15][0]=Kb0[15][0]+Kb1[15][0]+Kb2[15][0]+Kb3[15][0];
+  Kb[15][1]=Kb0[15][1]+Kb1[15][1]+Kb2[15][1]+Kb3[15][1];
+  Kb[15][5]=Kb0[15][5]+Kb1[15][5]+Kb2[15][5]+Kb3[15][5];
+  Kb[15][6]=Kb0[15][6]+Kb1[15][6]+Kb2[15][6]+Kb3[15][6];
+  Kb[15][10]=Kb0[15][10]+Kb1[15][10]+Kb2[15][10]+Kb3[15][10];
+  Kb[15][11]=Kb0[15][11]+Kb1[15][11]+Kb2[15][11]+Kb3[15][11];
+  Kb[15][15]=Kb0[15][15]+Kb1[15][15]+Kb2[15][15]+Kb3[15][15];
+  Kb[15][16]=Kb0[15][16]+Kb1[15][16]+Kb2[15][16]+Kb3[15][16];
+
+  Kb[16][0]=Kb0[16][0]+Kb1[16][0]+Kb2[16][0]+Kb3[16][0];
+  Kb[16][1]=Kb0[16][1]+Kb1[16][1]+Kb2[16][1]+Kb3[16][1];
+  Kb[16][5]=Kb0[16][5]+Kb1[16][5]+Kb2[16][5]+Kb3[16][5];
+  Kb[16][6]=Kb0[16][6]+Kb1[16][6]+Kb2[16][6]+Kb3[16][6];
+  Kb[16][10]=Kb0[16][10]+Kb1[16][10]+Kb2[16][10]+Kb3[16][10];
+  Kb[16][11]=Kb0[16][11]+Kb1[16][11]+Kb2[16][11]+Kb3[16][11];
+  Kb[16][15]=Kb0[16][15]+Kb1[16][15]+Kb2[16][15]+Kb3[16][15];
+  Kb[16][16]=Kb0[16][16]+Kb1[16][16]+Kb2[16][16]+Kb3[16][16];
```

Comment:
I know that it is quite messy to check. I just want to reduce the computation cost as low as possible by only calculate non-zero terms. Is it necessary to do that? Otherwise, I will try to put them in a loop. Thank you for your suggestion